llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -1372,8 +1372,11 @@ void RISCVInstrInfo::insertIndirectBranch(MachineBasicBlock &MBB,
                           .addMBB(&DestBB, RISCVII::MO_CALL);
 
   RS->enterBasicBlockEnd(MBB);
+  const TargetRegisterClass *RC = &RISCV::GPRRegClass;
+  if (STI.hasStdExtZicfilp())
```

Comment:
Doesn't this need to be based on the `cf-protection-branch` attribute? I thought the clang allows `cf-protection-branch` without Zicfilp?

Reply:
Ok